lib/v2/api.js

```diff
@@ -88,7 +88,7 @@
          * @param {Function} [cb(data)] Callback
          */
         apply: function(element, cb) {
-            var teamslug = element.getAttribute('data-gratipay-teamslug');
+            var projectslug = element.getAttribute('data-gratipay-teamslug');
```

Comment:
Seems like we could make this backwards-compatible here, ya? Use `-project-id` if present, otherwise `-team-slug`.

I think we should use `id` instead of `slug` to avoid breakage when slugs change (cf. https://github.com/gratipay/gratipay.com/issues/4373). And can we add the extra dash? ☺️